guides/configuration.md

```diff
@@ -0,0 +1,102 @@
+# Configuration
+
+When you first setup a Rails app with Solidus installed, you will want to customize its looks and behaviour to fit your company's needs. Solidus can be customized in six ways:
+
+* Configuration options
+* Overriding templates
+* Changing assets
+* Installing extensions
+* Custom service classes
+* Monkey-Patching Solidus
+* Run a private fork
+
+You will most probably use at least methods 1-4, and most shops we know also use some combination of the last three.
+
+As Solidus is an Engine, much of what the [Spree Guide on using Engines](http://guides.rubyonrails.org/engines.html) explains applies directly to Solidus.
+
+## Configuration options
+
+Since Solidus is a [Rails Engine](http://guides.rubyonrails.org/engines.html), much of its behaviour can be customized through the Solidus initializer. After you have run the `rails g spree:install` generator, this initializer will live in `config/initializers/spree.rb`. Open this file, and have a look at the first block (comments left out):
```

Comment:
Maybe explaining the `spree` vs. `solidus` incidence in the introductory section would be helpful to understand the differing file names especially for the unaware.